### ISSUE TITLE
flutter_gallery__transition_perf_e2e_ios32 failing

### ISSUE THREAD

**Issue by jmagman** [flutter/flutter#66162]
flutter_gallery__transition_perf_e2e_ios32 is failing in the devicelab:
```
2020-09-18T15:53:37.807818: stdout: flutter: > Grid@Material
2020-09-18T15:53:38.805892: stderr: Unhandled exception:
stderr: DriverError: Failed to fulfill RequestData due to remote error
2020-09-18T15:53:38.806006: stderr: Original error: JSON-RPC error -32000: Bad state: The client closed with pending request "ext.flutter.driver".
2020-09-18T15:53:38.806128: stderr: Original stack trace:
2020-09-18T15:53:38.806348: stderr: package:json_rpc_2/src/client.dart 123:62                       Client.sendRequest
stderr: package:json_rpc_2/src/peer.dart 98:15                          Peer.sendRequest
stderr: package:vm_service_client/src/scope.dart 64:23                  Scope.sendRequestRaw
2020-09-18T15:53:38.807074: stderr: package:vm_service_client/src/isolate.dart 361:19               VMIsolateRef.invokeExtension
stderr: package:flutter_driver/src/driver/vmservice_driver.dart 348:63  VMServiceFlutterDriver.sendCommand
stderr: package:flutter_driver/src/driver/driver.dart 503:45            FlutterDriver.requestData
stderr: package:e2e/e2e_driver.dart 73:42                               e2eDriver
stderr: ===== asynchronous gap ===========================
stderr: test_driver/transitions_perf_e2e_test.dart 7:24                 main
stderr: dart:isolate-patch/isolate_patch.dart 301:19                    _startIsolate.<fn>
stderr: dart:isolate-patch/isolate_patch.dart 168:12                    _RawReceivePortImpl._handleMessage
```
[flutter_gallery__transition_perf_e2e_ios32_5c85803_2.log](https://github.com/flutter/flutter/files/5248934/flutter_gallery__transition_perf_e2e_ios32_5c85803_2.log)

**Comment by jmagman** [comment]
Suspect https://github.com/flutter/flutter/commit/409e994cd03749d5a903f01cf4ec5b3ee074a2ee, will revert to see.

**Comment by jmagman** [comment]
Or not, looks like https://github.com/flutter/flutter/pull/66123 fixed a different test failure.

**Comment by bkonyi** [comment]
Yes, this was a rather involved change that required various fixes over multiple commits. If at all possible, we should try and avoid reverting these commits and work on finding the cause of the issue or disable DDS for these tests.

**Comment by zanderso** [comment]
Sorry, our tree hygiene policy means that we have to revert in situations like this. See https://github.com/flutter/flutter/wiki/Tree-hygiene for details.

I landed the revert here https://github.com/flutter/flutter/pull/66164, but the test continues to fail, so I filed an infra issue here https://github.com/flutter/flutter/issues/66166. There continue to be the same warning messages from DDS, but I'm not able to determine that that is the reason the test is failing.
